src/Domain/Bootstrap.php

```diff
@@ -134,6 +139,18 @@ function() {
 		$this->container->get( TasksController::class )->init();
 		$this->container->get( JetpackWooCommerceAnalytics::class )->init();
 
+		try {
```

Comment:
I understand this is temporary but wouldn't it be better to gate this based on class existence either here or within the `init` method?